README.md

```diff
@@ -1,30 +1,57 @@
+<img src="screenshots/name.png" width="200"> <img src="screenshots/record_yourself.png" width="200"> <img src="screenshots/recording.png" width="200">
 
-### User support:
-Users will have to login with the Google built-in authentication in order to use the app. Once they’re logged in they can fill their profile with basic information about themselves, interests and voice memos to present themselves.
+### Match
+Swipe right when yoy hear someone you like, swipe left otherwise.
 
-### Local cache:
-The messages the user sends to other users are cached in the app.
-A list of matched profiles is also stored and a limited number of non matched profiles for offline use too.
+<img src="screenshots/match.png" width="200"> 
 
-### Offline mode:
-While offline, the user will be able to match the profiles in cache, view previously matched profiles and see their conversations. They will also be able to write new messages that will be sent once a network connection is available.
+### Your matches
+We grouped all the person you've matched with in the same place, this way you'll be able to easily find them, have a closer look at their profile, see what they like,
+talk to them, and once you've plan a date, easily find them in real life thanks to a map that shows the location of both participants (if they are logged in).
+
+<img src="screenshots/my_matches.png" width="200"> <img src="screenshots/profile_match.png" width="200"> <img src="screenshots/chat.png" width="200"> <img src="screenshots/map.png" width="200"> 
+
+### Plan a date
+The weather is directly accessible from the app, so you can plan for the most perfect date and not forget about it by directly creating an event in your calendar!
+
+<img src="screenshots/weather.png" width="200"> <img src="screenshots/calendar.png" width="200"> 
+
+### Profile
+Edit your informations and preferences, so that potential matches we present you with may fit your expectations better.
+
+<img src="screenshots/profile_page.png" width="200"> <img src="screenshots/edit_info.png" width="200"> 
+
+## Configuration:
+The first two steps are required for the app to work (the login and the OpenWeather API key), the third one is optional.
+
+### Login 
+To login into our app, we need to add the Debug variant of your SHA1 into the Firebase console of our project. To do this, go into the folder where the project has been cloned
+and open a terminal, enter:
+./gradlew signingReport
+
+### OpenWeather API Key
+You'll first need to generate an OpenWeather API key. In the local.properties file at the root of the folder add the following line:
+OPEN_WEATHER_MAP_KEY="YOUR_API_KEY"
+
+### Facebook login (optional)
+If you wish to login with Facebook you will have to follow the step 6 of the [Facebook Login for Android - Quickstart](https://developers.facebook.com/docs/facebook-login/android)
+ 

```

Comment:
Can you add  this phrase here? -> Once you have generated your development key, you can reach us. Then we add your key in Facebook Developers and you'll be able to use Facebook to login.

Reply:
Yep!